src/util.js

```diff
@@ -452,10 +452,10 @@ export default {
   },
 
   getGlobal: function() {
-    if (typeof window !== 'undefined') {
-      return window;
-    } else if (typeof self !== 'undefined') {
+    if (typeof self !== 'undefined') {
       return self;
+    } else if (typeof window !== 'undefined') {
+      return window;
```

Comment:
On all of them: https://github.com/openpgpjs/openpgpjs/blob/master/src/worker/worker.js#L20

Preferring self over window fixed this.